runner/runners/invoke.go

```diff
@@ -259,7 +279,11 @@ func (inv *Invoker) run(cmd *runner.Command, id runner.RunID, abortCh chan struc
 				parentDir, _ := filepath.Split(co.Path() + "/")
 				err = setupJDKSymlink(parentDir, pp.GetValue())
 				if err != nil {
-					return runner.FailedStatus(id, err, tags.LogTags{JobID: cmd.JobID, TaskID: cmd.TaskID, Tag: cmd.Tag})
+					msg := fmt.Sprintf("Failed setting up JDK symlink to %s: %s", pp.GetValue(), err)
+					failedStatus := runner.FailedStatus(id, errors.New(msg),
+						tags.LogTags{JobID: cmd.JobID, TaskID: cmd.TaskID, Tag: cmd.Tag})
+					failedStatus.ActionResult = &bazelapi.ActionResult{GRPCStatus: getInternalErrorStatus(msg)}
```

Comment:
think this one needs an `if runType = runner.RunTypeBazel` too

Reply:
this one's OK, in a runtype bazel code path already